test/unit/object/conftest.py

```diff
@@ -271,11 +271,28 @@ def if_none_match_header(etag):
     return {'If-None-Match': etag} if etag is not None else None
 
 
+@pytest.fixture()
+def if_match_sha1_header(etag, sha1):
+    headers = {}
+    if etag is not None:
+        headers['If-Match'] = etag
+    if sha1 is not None:
+        headers['Content-MD5'] = sha1
+    if not headers:
+        headers = None
+    return headers
```

Comment:
ditto from above, we could do `return headers or None` and skip line 281/282

Reply:
I'm okay to duplicate this for the tests, was thinking it could be nice to share between file/folder